### ISSUE TITLE
When trying to render page I get ERROR { Error: Your middleware did not return anything to the user. This normally happens when no route was matched or if the router did not reply with anything. 

### ISSUE THREAD

**Issue by AdamCollins** [franciscop/server#118]
I am trying to render a simple EJS page(currently doesn't contain any variables) when a user goes to the route directory. The page _does_ render but I get the warning in my title.
```javascript
const server = require('server');
const { get, error } = server.router;
const { render, status} = server.reply;

server([
    get('/',ctx => render('index.ejs'))
]);
```
From reading this issue[here](https://github.com/franciscop/server/issues/92) I thought I may be missing some sort of error handling so I tried adding
```javascript
error(ctx => status(500).send(ctx.error.message))
``` 
to my ``server`` function but with no prevail.

Also, I noticed it does work but if I change my the ``server`` function to this
```javascript
server(ctx => render('index.ejs'));
```
but obviously I would like to have more than one route.

Any help would be appreciated!

**Environment (optional):**
 - OS: Ubuntu 18.04.2 LTS
 - Node.js: 8.10
 - Server.js:LTS

**Comment by franciscop** [comment]
Thanks for the complete bug report, I'll try to reproduce later. This does seems like a server.js bug.

**Comment by franciscop** [comment]
Hi @AdamCollins, unfortunately I couldn't reproduce the issue and it works fine for me. Could you please put your `package.json` and copy/paste the full error? I suspect one of these 3 issues might be happening:

- EJS is not installed ([it has to be installed manually](https://serverjs.io/documentation/reply/#render-)) -> `npm install ejs`. This will make the route throw an error, so it will not render anything and so you'll get this issue.
- The EJS template is not in the `views` folder, which will make an error similar to the one above. This can be changed [with the `views` option](https://serverjs.io/documentation/options/#views) if you want.
- Your browser is requesting a different resource like `favicon.ico`, which will not be found. To debug this, add an *extra* route that does this to see if there's any extra request that you are not intending to make:

```js
const server = require("server");
const { get, error } = server.router;
const { render, status } = server.reply;

server([
  ctx => {
    console.log(ctx.method, ctx.url);
  },
  get("/", ctx => render("index.ejs"))
]);
```

There might be some other issue, but these 3 are the ones I'd check for first. I [created a repository](https://github.com/franciscop/server-ejs-example) with working code. Is there anything different from your code?

**Comment by franciscop** [comment]
BTW I strongly believe it is the 3rd option of the browser asking for a resource that cannot be found, because it's the only one that would account for the situation you are seeing that `server(get(ctx => ...))` does not work but `server(ctx => ...)` works.

**Comment by AdamCollins** [comment]
The issue was option 3. After adding the references to the public folder the warning disappears. 
Thank you!

**Comment by franciscop** [comment]
Hey @AdamCollins, so I also went ahead and just published `1.0.19` fixing the issue on server.js side. Now the error message will be much more clear like this:

```
Your middleware did not return anything for this request:

GET /abc.png

This normally happens when no route was matched or if the router did not reply with anything. Make sure to return something, even if it's a catch-all error.

Documentation for reply: https://serverjs.io/documentation/reply/
Relevant issue: https://github.com/franciscop/server/issues/118
```

Please feel free to open any other issue you find 😄
